modules/create_timepoint/php/timepoint.class.inc

```diff
@@ -90,9 +90,23 @@ class Timepoint extends \NDB_Page implements ETagCalculator
         );
 
         // Retrieve visit labels.
-        $visit_options = [];
+        $projectID = $candidate->getProjectID();
+        // Visits associated with the candidate's project
+        $visitsForProject = \Project::getVisitList($projectID);
```

Comment:
why not use https://github.com/aces/Loris/blob/main/php/libraries/Utility.class.inc#L228 instead ?

(I agree the utility class function is less than ideal but an upcoming PR should be cleaning that up and the addition of another function in the project class at this stage might confuse things)